src/Umbraco.Web.UI.Client/src/common/services/minieditorhelper.service.js

```diff
@@ -0,0 +1,79 @@
+(function () {
+    'use strict';
+
+    function miniEditorHelper(dialogService, editorState, fileManager, contentEditingHelper) {
+
+        var launched = false;
+
+        function launchMiniEditor(node) {
```

Comment:
OK yes makes sense to make it public in our docs - after seeing the current state of the implementation.